src/app/components/nodes-view/nodes-view.component.ts

```diff
@@ -111,7 +111,7 @@ export class NodesViewComponent implements OnInit {
     }
   }
 
-  onPaginatorChanged(page: PageEvent) {
+  onPaginatorChanged() {
```

Comment:
hi @akhilpb001 

The rest of the changes are straightforward, but for this one. I just want to make sure this is intentional, is it?

Reply:
Yes, its unused code, hence removed.

Reply:
If we are getting travis precommit stage green, that means we are good to commit the code.

Reply:
Got it, thanks